kong/plugins/key-auth/handler.lua

```diff
@@ -17,13 +17,20 @@ local KeyAuthHandler = BasePlugin:extend()
 
 KeyAuthHandler.PRIORITY = 1000
 
+local OPTIONS = "OPTIONS"
```

Comment:
:+1:, we were following the example over here: https://github.com/Mashape/kong/blob/master/kong/plugins/cors/handler.lua#L8